### ISSUE TITLE
Crash: Metadata short title

### ISSUE THREAD

**Issue by messersc** [bihealth/scelvis#27]
```
  File "/app/scelvis/ui/main.py", line 121, in render_children_goto
    meta.short_title, id="menu-item-%s" % meta.id, href="/dash/viz/%s" % meta.id
AttributeError: 'Data' object has no attribute 'short_title'
```

**Comment by holtgrewe** [comment]
Which version?

**Comment by holtgrewe** [comment]
Fixed in `master` and `v0.4.1`.
